### DISCUSSION TITLE
Updating Alpine version on LTS tagged image

### DISCUSSION THREAD

**Discussion by jakauppila** [renovatebot/renovate#29744] [Request Help]
### What would you like help with?

I would like help with my configuration

### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us which platform (GitHub, GitLab, etc) and which version of Renovate.

Bitbucket Data Center

### Please tell us more about your question or problem

I'm trying to configure Renovate to keep our PowerShell container up to date on LTS releases. We have the following FROM statement:

`FROM mcr.microsoft.com/powershell:lts-alpine-3.16@sha256:1c5f407d7ac27e079a4f620e31795cb445abcae6fefc36e5f018886b47a2c380`

And are attempting to use the following packageRules configuration:

```
"packageRules": [
    {
      "matchDatasources": [
        "docker"
      ],
      "matchManagers": [
        "dockerfile"
      ],
      "matchFileNames": [
        "7/alpine/Dockerfile"
      ],
      "versioning": "loose",
      "allowedVersions": "/-alpine-/"
    }
  ]
```

I have tried a number of variations based on related Discussions here, but I have not been able to get it to upgrade from `lts-alpine-3.16` to `lts-alpine-3.17`.

### Logs (if relevant)

<details><summary>Logs</summary>

```

DEBUG: Dependency mcr.microsoft.com/powershell has unsupported/unversioned value lts-alpine-3.16 (versioning=loose) (repository=USER/powershell)
DEBUG: packageFiles with updates (repository=USER/powershell, baseBranch=master)
        "config": {
          "dockerfile": [
            {
              "deps": [
                {
                  "depName": "mcr.microsoft.com/powershell",
                  "currentValue": "lts-alpine-3.16",
                  "currentDigest": "sha256:1c5f407d7ac27e079a4f620e31795cb445abcae6fefc36e5f018886b47a2c380",
                  "replaceString": "mcr.microsoft.com/powershell:lts-alpine-3.16@sha256:1c5f407d7ac27e079a4f620e31795cb445abcae6fefc36e5f018886b47a2c380",
                  "autoReplaceStringTemplate": "{{depName}}{{#if newValue}}:{{newValue}}{{/if}}{{#if newDigest}}@{{newDigest}}{{/if}}",
                  "datasource": "docker",
                  "depType": "final",
                  "updates": [],
                  "packageName": "mcr.microsoft.com/powershell",
                  "versioning": "loose",
                  "warnings": []
                }
              ],
              "packageFile": "7/alpine/Dockerfile"
            }

```

</details>

**Comment by rarkins** [comment]
You could try using `extractVersion` to separate out the `lts-alpine-` part as compatibility and `3.16` as version. You can then keep using `docker` versioning without any `allowedVersions`.
